docs/howto/configure.rst

```diff
@@ -0,0 +1,10 @@
+Configure database connection settings
+--------------------------------------
+
+Ensure that your database connection settings are correctly configured.
+
+ - Either your environment variables `PGHOST`, `PGPORT` and `PGUSER` are properly set.
+ - Or you can set the environment variables `SEPTENTRION_HOST`, `SEPTENTRION_PORT`, `SEPTENTRION_USERNAME`.
+ - configuration file `septentrion.ini`
```

Comment:
I feel like this bullet point is not done :)

Reply:
WIP commit is WIP indeed :D